app/internal/proxy/hijack_responsefilter_disjoint_test.go

```
@@ -0,0 +1,120 @@
+		"/exec/e1/start",
+		"/libpod/containers/c1/attach",
+		"/libpod/exec/e1/start",
+	}
+
+	for _, path := range hijackPaths {
+		t.Run(path, func(t *testing.T) {
+			if !filter.IsHijackCandidatePath(http.MethodPost, filter.NormalizePath(path)) {
+				t.Fatalf("IsHijackCandidatePath(POST, %q) = false; the hijack candidate set changed and this test's fixture list is stale", path)
+			}
+
+			resp := jsonResponse(t, path, redactableBody)
```

golangci-lint (2.12.2)

[error] response body must be closed

(bodyclose)